container/go/cmd/puller/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
license header in all files.

Reply:
You can just copy this one https://github.com/bazelbuild/rules_docker/blob/master/container/build_tar.py